atd-vzd/views/view_fatalities.sql

```diff
@@ -0,0 +1,32 @@
+-- view of fatalities in jurisdiction ID 5 (austin full purpose) with crash attributes joined
+CREATE OR REPLACE VIEW view_fatalities AS (
+    SELECT
+        f.id,
+        f.crash_id,
+        f.victim_name,
+        f.ytd_fatal_crash,
+        f.ytd_fatality,
+        f.person_id,
+        f.primaryperson_id,
+        TO_CHAR(crashes.crash_date,
+            'yyyy') AS year,
+        CONCAT_WS(' ',
```

Comment:
TIL `CONCAT_WS()`! I have needed this about a billion times and always did a messy `CONTACT()`. This is great.

Reply:
yeah i was stoked to find this function it made my life a lot easier!